lib/ansible/plugins/action/vyos.py

```diff
@@ -38,7 +38,7 @@ class ActionModule(ActionNetworkModule):
     def run(self, tmp=None, task_vars=None):
         del tmp  # tmp no longer has any effect
 
-        self._config_module = True if self._task.action == 'vyos_config' else False
+        self._config_module = True if 'vyos_config' in self._task.action else False
```

Comment:
I'd rather see something like
```suggestion
        module_name = self._task.action.split('.')[-1]
        self._config_module = True if module_name == 'vyos_config' else False
```
So that we're checking the whole module name. Otherwise if someone has a custom module named, e.g. `vyos_configuration`, unexpected things might happen.